scripts/oneoff/inject-framework-dates.py

```diff
@@ -26,20 +26,23 @@
         'applicationsCloseAtUTC': '2016-01-19T15:00:00Z',
         'intentionToAwardAtUTC': '2016-02-18T12:00:00Z',
         'frameworkLiveAtUTC': '2016-03-03T12:00:00Z',
+        'frameworkExpiresAtUTC': '1970-01-01T00:00:00Z',
     },
     'digital-outcomes-and-specialists-2': {
         'clarificationsCloseAtUTC': '2016-12-01T17:00:00Z',
         'clarificationsPublishAtUTC': '2016-12-07T17:00:00Z',
         'applicationsCloseAtUTC': '2016-12-14T17:00:00Z',
         'intentionToAwardAtUTC': '2017-01-16T12:00:00Z',
         'frameworkLiveAtUTC': '2017-01-27T12:00:00Z',
+        'frameworkExpiresAtUTC': '2018-01-29T12:00:00Z',
     },
     'digital-outcomes-and-specialists-3': {
         'clarificationsCloseAtUTC': '2525-01-01T12:00:00Z',
         'clarificationsPublishAtUTC': '2525-01-01T12:00:00Z',
         'applicationsCloseAtUTC': '2525-01-01T12:00:00Z',
         'intentionToAwardAtUTC': '2525-01-01T12:00:00Z',
         'frameworkLiveAtUTC': '2525-01-01T12:00:00Z',
+        'frameworkExpiresAtUTC': '1970-01-01T00:00:00Z',
```

Comment:
If we're doing dummy dates for now it might make sense to match with the other dummy dates here?

Reply:
Yep, sounds reasonable.